### PR TITLE
Feat/add dummy rows

### PR DESCRIPTION
**Pull request by seonim-ryu**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

#### Feature
* Create dummy rows on viewport when setting `showDummyRows`.
```js
gridOptions = {
  showDummyRows: true // default false
}
```

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/view/bodyRows.tsx [hunk 1]
```diff
@@ -12,32 +13,57 @@ interface OwnProps {
...
+  }: {
+    dummyRowCount: number;
+    columnNames: string[];
+    startIndex: number;
+  }) {
+    const dummyRows = [];
+
+    for (let i = 0; i < dummyRowCount; i += 1) {
+      dummyRows.push(
+        <BodyDummyRow key={String(i)} index={startIndex + i} columnNames={columnNames} />
```
Comment (jung-han) [inline src/view/bodyRows.tsx]: key에 toString을 해주지 않아도 될 것 같습니다.

File: src/view/bodyRows.tsx [hunk 2]
```diff
@@ -12,32 +13,57 @@ interface OwnProps {
...
   }
 
-  public render({ rows, columns }: Props) {
+  public render({ rows, columns, dummyRowCount }: Props) {
     const columnNames = columns.map(({ name }) => name);
 
     return (
       <tbody>
         {rows.map((row) => (
           <BodyRow key={String(row.rowKey)} viewRow={row} columnNames={columnNames} />
```
Comment (jung-han) [inline src/view/bodyRows.tsx]: 여기도 key에 toString을 해주지 않아도 될 것 같습니다.

File: src/view/bodyRows.tsx [hunk 3]
```diff
@@ -18,22 +19,44 @@ interface StoreProps {
...
         ))}
-        {dummyRowCount ? <div /> : null}
+        {dummyRowCount
+          ? this.getDummyRows({ dummyRowCount, columnNames, startIndex: rows.length })
+          : null}
```
Comment (js87zz) [inline src/view/bodyRows.tsx]: 아래처럼 좀 더 줄일 수 있을 듯 합니다
```
!!dummyRowCount
          && this.getDummyRows({ dummyRowCount, columnNames, startIndex: rows.length })
```

**Comment by js87zz** [comment]
리뷰 완료입니다!고생하셨습니다!

File: src/view/bodyRows.tsx (hunk 1 above)
Comment (seonim-ryu) [reply src/view/bodyRows.tsx]: 그리고 `i` 네이밍이 별로라서.. `index`로 변경해야겠네요ㅎ

File: src/view/bodyRows.tsx (hunk 3 above)
Comment (seonim-ryu) [reply src/view/bodyRows.tsx]: 오우 그렇네요 감사합니다ㅎ

**Pull request merged**